src/alertsmanagement/azext_alertsmanagement/custom.py

```diff
@@ -0,0 +1,324 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-locals
+# pylint: disable=unused-argument
+# pylint: disable=too-many-boolean-expressions
+import json
+
+from knack.util import CLIError
+
+
+def list_alertsmanagement_operation(cmd, client):
+    return client.list()
+
+
+def change_state_alertsmanagement_alert(cmd, client,
+                                        alert_id,
+                                        new_state=None):
+    return client.change_state(alert_id=alert_id, new_state=new_state)
+
+
+def meta_data_alertsmanagement_alert(cmd, client,
+                                     identifier=None):
+    return client.meta_data(identifier=identifier)
+
+
+def get_all_alertsmanagement_alert(cmd, client,
+                                   target_resource=None,
+                                   target_resource_type=None,
+                                   target_resource_group=None,
+                                   monitor_service=None,
+                                   monitor_condition=None,
+                                   severity=None,
+                                   alert_state=None,
+                                   alert_rule=None,
+                                   smart_group_id=None,
+                                   include_context=None,
+                                   include_egress_config=None,
+                                   page_count=None,
+                                   sort_by=None,
+                                   sort_order=None,
+                                   select=None,
+                                   time_range=None,
+                                   custom_time_range=None):
+    return client.get_all(target_resource=target_resource, target_resource_type=target_resource_type, target_resource_group=target_resource_group, monitor_service=monitor_service, monitor_condition=monitor_condition, severity=severity, alert_state=alert_state, alert_rule=alert_rule, smart_group_id=smart_group_id, include_context=include_context, include_egress_config=include_egress_config, page_count=page_count, sort_by=sort_by, sort_order=sort_order, select=select, time_range=time_range, custom_time_range=custom_time_range)
```

Comment:
Without a 30 inch 4k monitor, it's hard for me to read this incredibly long line.

Reply:
Done